sdks/java/io/jdbc/pom.xml

```diff
@@ -320,6 +315,11 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
Again the same for `slf4j-api`: it's not a test dependency, it's a compile one as SLF4J `Logger` is used in the IO.